File: LNPG_02_ESTRUTURAS_DE_CONTROLE/ATV_3/HASKELL/main.hs
main::IO()
main = do 
    putStrLn "Digite a quantidade de jogos vendidos: "
    input <- getLine
    let games = read input :: Double

    let gameValue = 19.9
    let totalProfit = gameValue*games
    let partSalary = totalProfit/2
    let totalSalary = fromIntegral (floor (games / 15)) * (totalProfit * 0.08) + partSalary
    putStrLn ("Total arrecadado: " ++ show (totalProfit))
    putStrLn ("Salario sem bonus: " ++ show (partSalary))
    putStrLn ("Com bonus: " ++ show (totalSalary))

File: LNPG_02_ESTRUTURAS_DE_CONTROLE/ATV_3/C/main.c
#include <stdio.h>
#include <math.h>

int main()
{
    int games;

    printf("Digite a quantidade de jogos vendidos: ");
    scanf("%d", &games);

    double gameValue = 19.9;
    double totalProfit = gameValue * games;
    double partSalary = totalProfit / 2;
    double totalSalary = floor(games / 15) * (totalProfit * 0.08) + partSalary;

    printf("Total arrecadado: %.2f\nSalario sem bonus: %.2f\nCom bonus: %.2f\n", totalProfit, partSalary, totalSalary);

    return 0;
} 